src/components/form/AddPlant.tsx

```diff
@@ -0,0 +1,128 @@
+import { ChangeEvent, FormEvent, useContext, useState } from 'react';
+import { plantBotanicNames } from '../../data/plantBotanicNames';
+import { rooms } from '../../data/rooms';
+import sprite from '../../icons/symbol-defs.svg';
+import { PlantsList } from '../App';
+
+type AddPlantProps = {
```

Comment:
i usually just call this type Props
its not like you gonna use it elswhere, so there is no need for unique naming
also better use interface